gunicorn/workers/__init__.py

```diff
@@ -17,6 +17,6 @@
 }
 
 
-if sys.version_info >= (3, 3):
+if sys.version_info >= (3, 4):
     # gaiohttp worker can be used with Python 3.3+ only.
```

Comment:
`3.3+` -> `3.4+`?

Reply:
I did this change to match this: https://github.com/benoitc/gunicorn/blob/master/gunicorn/workers/gaiohttp.py#L10

Reply:
I mean the comment `# gaiohttp worker can be used with Python 3.3+ only.` should be updated to `# gaiohttp worker can be used with Python 3.4+ only.`